charts/delphai-keda/templates/deployment.yml

```diff
@@ -71,3 +72,68 @@ spec:
               containerPort: {{ .Values.http.port }}
             - name: h2c
               containerPort: {{ .Values.grpc.port }}
+
+        {{ if and .Values.http.enabled .Values.http.public .Values.http.publicWithOAuth }}
+        - name: oauth2-proxy
+          image: "quay.io/oauth2-proxy/oauth2-proxy:v7.2.0"
+          args:
+            - '--email-domain=*'
```

Comment:
This should work being passed as ENV, but it doesn't

Reply:
:(

Reply:
Yeah, too a lot of time to debug that — all other params were working fine